pinax/stripe/tests/test_actions.py

```diff
@@ -3089,26 +3092,34 @@ def setUp(self):
         "disabled_reason":null
       }
     }""")
-        self.not_custom_account_data = json.loads(
+        cls.not_custom_account_data = json.loads(
             """{
-      "support_phone":"7788188181",
+      "business_logo":null,
       "business_name":"Woop Woop",
       "business_url":"https://www.someurl.com",
-      "support_url":"https://support.someurl.com",
-      "country":"CA",
-      "object":"account",
-      "business_logo":null,
       "charges_enabled":true,
-      "support_email":"support@someurl.com",
+      "country":"CA",
+      "default_currency":"cad",
       "details_submitted":true,
+      "display_name":"Some Company",
       "email":"operations@someurl.com",
-      "transfers_enabled":true,
```

Comment:
It fails because `"transfers_enabled"` is expected to be there, but is not.
Can you shed some light, @lukeburden ?

Reply:
https://stripe.com/docs/upgrades#2017-04-06

it has been renamed to payouts_enabled

Reply:
Another migration is on its way

Reply:
Nice find.
Full docs: https://stripe.com/docs/transfer-payout-split (the renaming is mentioned at the end).